jeandle-docs/getting-started.md

```diff
@@ -1,6 +1,6 @@
 # Getting Started
 ## Getting the Source Code and Building
-Jeandle is composed of two separate code repositories: [jeandle-llvm](https://github.com/jeandle/jeandle-jdk) and [jeandle-jdk](https://github.com/jeandle/jeandle-llvm). Both repositories need to be built individually, following the same process as their upstream repositories. Note that when building jeandle-jdk, you must specify the installation directory of jeandle-llvm using the option ```--with-jeandle-llvm=<directory>```.
+Jeandle is composed of two separate code repositories: [jeandle-llvm](https://github.com/jeandle/jeandle-jdk) and [jeandle-jdk](https://github.com/jeandle/jeandle-llvm). Both repositories need to be built individually, following the same process as their upstream repositories. Note that when building jeandle-jdk, you must specify the installation directory of jeandle-llvm using the option ```--with-jeandle-llvm=<directory>```. When cross-compiling, you must specify the installation directory of jeandle-llvm of the host environment using the option ```--with-host-jeandle-llvm=<directory>```.
```

Comment:
```suggestion
Jeandle is composed of two separate code repositories: [jeandle-llvm](https://github.com/jeandle/jeandle-jdk) and [jeandle-jdk](https://github.com/jeandle/jeandle-llvm). Both repositories need to be built individually, following the same process as their upstream repositories. Note that when building jeandle-jdk, you must specify the installation directory of jeandle-llvm using the option ```--with-jeandle-llvm=<directory>```. When cross-compiling, you must specify the installation directory of jeandle-llvm of the target system using the option ```--with-host-jeandle-llvm=<directory>```.
```
According to the openjdk document, maybe `target` is a more understandable description.